### ISSUE TITLE
legend dpi + maxWidth doesnt work anymore

### ISSUE THREAD

**Issue by ochriste** [mapfish/mapfish-print#553]
when specifying dpi + maxWidth in the Legend bloc, the getLegendGraphic should include the dpi so we can change the generated image dimensions

this doesnt seem to work anymore in 3.6

**Comment by gnerred** [comment]
Very annoying issue for us :O(...

**Comment by cmangeat** [comment]
https://github.com/mapfish/mapfish-print/issues/554 works.
Don't understand what are the differences between 3.6.0 (fails) and 3.5.0 (works).

**Comment by gnerred** [comment]
is the problem solved with #554 ?

**Comment by sbrunner** [comment]
Yes it should :-)

**Comment by gnerred** [comment]
Thanks. Will this be automatically integrated in next GMF 2.1.5 ?

**Comment by sbrunner** [comment]
No, it will be integrate in version 2.3.x, but you can change the version in the CONST_print_url
 (be careful on all the update...)

**Comment by gnerred** [comment]
What do I have to be careful to ? What is the new version number ?

If I try with the new version and it goes bad, can I just put again actual version 3.6 and all goes back as before ?

**Comment by sbrunner** [comment]
No, every-time you do an update you should apply your changes again because it's a CONST file :-)

**Comment by gnerred** [comment]
ah ok... and what is the version number I have to use instead of 3.6 ?

**Comment by sbrunner** [comment]
The next one, 3.10... @pvalsecc can you create this release?

**Comment by gnerred** [comment]
When do you plan to create this release ? Thanks

**Comment by gnerred** [comment]
@sbrunner , any idea ? This issue is a bit annoying for us, I get several users negative feedback about it...

**Comment by sbrunner** [comment]
@pvalsecc up

**Comment by gnerred** [comment]
@sbrunner , there is still no new release of Mapfish-print... any idea of the delay ? Thanks

**Comment by pvalsecc** [comment]
@gnerred , as soon as I can merge #567 and #565, I release 3.10.0. Should be in the next couple of days.
